### PR TITLE
CCCT-533- ui icon changes as per the new figma

### PR DESCRIPTION
**Pull request by pm-dimagi**
https://dimagi.atlassian.net/browse/CCCT-533

The ui changes 
-- Icon update
-- commcare logo update 

## Safety Assurance

- [ ] If the PR is high risk, "High Risk" label is set
- [ ] I have confidence that this PR will not introduce a regression for the reasons below
- [ ] Do we need to enhance manual QA test coverage ? If yes, "QA Note" label is set correctly

### Safety story
 

Release Note: Updated Iconography and Logo

### CODE REVIEW

**Review by shubham1g5 — CHANGES_REQUESTED** [review]
The code changes looks fine. 

1. Is it possible to add screenshots in the PR on how it looks in the app ?
2. Is it possible to use vector drawables insted of png to take into account different screen densities.

**Comment by pm-dimagi** [comment]
WhatsApp Image 2024-11-12 at 11 34 50 (1)
WhatsApp Image 2024-11-12 at 11 34 50 (2)
WhatsApp Image 2024-11-12 at 11 34 50

**Review by avazirna — COMMENTED** [review]
@pm-dimagi can you include a screenshot of the full Home screen, with the `Incomplete` and `Saved` buttons?

File: app/res/layout/install_confirm_fragment.xml [hunk 1]
```diff
@@ -92,8 +75,9 @@
                                 android:id="@+id/btn_start_install"
                                 android:layout_width="wrap_content"
                                 android:layout_height="wrap_content"
+                                android:padding="16dp"
                                 SquareButtonWithText:backgroundColor="@color/cc_attention_positive_color"
-                                SquareButtonWithText:img="@drawable/install_start"
+                                SquareButtonWithText:img="@drawable/download"
```
[the remark this answers is not in the document]
Comment (pm-dimagi) [reply app/res/layout/install_confirm_fragment.xml:80]: I was not aware of its usage at other places so i kept the old one and added the new ones

**Review by shubham1g5 — APPROVED** [review]
Changes looks good to me. It's a good idea to add a release note to any big changes like this and also tick the safety checklist in PR template.

**Review by avazirna — APPROVED** [review]

**Comment by avazirna** [comment]
@damagatchi retest this please

**Comment by kcowger** [comment]
@pm-dimagi I'm seeing a different design (different icons, different icon sizes) on the final assets provided to SaaS here: https://www.figma.com/design/UWjp1wRNAO0o9q2llwDKMc/CommCare-Cosmetic-Changes-Post-Nov-'22Rebrand?node-id=111-2&node-type=canvas

Can you confirm where you got these icons and icon sizes? I want to make sure we are working on off of the same assets.

**Comment by pm-dimagi** [comment]
@kcowger i downloaded the assets from the figma file only not got any icon sizes
I got this figma file for refrence https://www.figma.com/design/UWjp1wRNAO0o9q2llwDKMc/CommCare-Cosmetic-Changes-Post-Nov-'22Rebrand?node-id=625-2&node-type=canvas&t=ijXRvUyGF2lRrV6G-0

**Review by shubham1g5 — APPROVED** [review]
Changes looks good, but tests seems to be failing - 

````
org.commcare.android.tests.application.AppUpdateTest > testAppUpdate FAILED
21:09:01     java.lang.ExceptionInInitializerError at AppUpdateTest.kt:197
21:09:01         Caused by: java.lang.IllegalStateException at AppUpdateTest.kt:197
21:09:01 
21:09:01 org.commcare.android.tests.application.AppUpdateTest > testUpdateToBrokenApp FAILED
21:09:01     java.lang.NoClassDefFoundError at AppUpdateTest.kt:197
21:09:02 
21:09:02 org.commcare.android.tests.application.AppUpdateTest > testAppIsUpToDate FAILED
21:09:02     java.lang.NoClassDefFoundError at AppUpdateTest.kt:197
21:09:03 
21:09:03 org.commcare.android.tests.application.AppUpdateTest > testAppUpdateWithSuiteFixture FAILED
21:09:03     java.lang.NoClassDefFoundError at AppUpdateTest.kt:197
21:09:03 
21:09:03 org.commcare.android.tests.application.AppUpdateTest > testUpdateToAppWithIncompatibleVersion FAILED
21:09:03     java.lang.NoClassDefFoundError at AppUpdateTest.kt:197
21:09:04 
21:09:04 org.commcare.android.tests.application.AppUpdateTest > testUpdateToAppWithMultimedia FAILED
21:09:04     java.lang.NoClassDefFoundError at AppUpdateTest.kt:197
````

**Comment by avazirna** [comment]
>  some revamp in colors

@pm-dimagi from this screenshot, it seems that the app bar colour was not updated according with the new design, will that be address in a separate PR?

**Comment by pm-dimagi** [comment]
Yess

**Comment by pm-dimagi** [comment]
@damagatchi please retest this

**Comment by OrangeAndGreen** [comment]
@damagatchi retest this please

**Review by OrangeAndGreen — APPROVED** [review]

**Comment by pm-dimagi** [comment]
@damagatchi retest this please

**Comment by pm-dimagi** [comment]
@damagatchi retest this please

**Comment by shubham1g5** [comment]
@pm-dimagi FYI that In your last screenshot, the text on the log out button is getting cut off.

**Pull request merged**